pkg/scheduler/factory.go

```diff
@@ -480,40 +473,17 @@ func MakeDefaultErrorFunc(client clientset.Interface, podQueue internalqueue.Sch
 			klog.Errorf("Error scheduling %v/%v: %v; retrying", pod.Namespace, pod.Name, err)
 		}
 
+		if !podQueue.IsPodBeingProcessed(podInfo.Pod.UID) {
+			klog.Warningf("Pod %v/%v may have been deleted.", pod.Namespace, pod.Name)
+			return
+		}
+
+		// TODO(Huang-Wei): cleanup schedulingCycle.
```

Comment:
I'm not sure I follow the TODO.

Reply:
`schedulingCycle` was introduced to handle the racing issue between the async goroutine with main scheduler goroutine (`scheduleOne()`), maybe we can cleanup `schedulingCycle` after this PR gets merged. But as the binding cycle is still async, so maybe not possible yet. Nvm, let me remove this comment.